src/reducers/utils.js

```diff
@@ -160,6 +160,10 @@ export const reduceUpdateData = (state, action) => {
   return state.mergeDeepIn([myId], data)
 }
 
+export const isNullTimeStamp = timestamp => {
```

Comment:
Does `{"T":0,"NT":0}` happen?

If so, need to confirm with @chhsiao1981 as it should always return value.

Reply:
It's possible in LastSeen

Reply:
This is for CommentCreateTS, but got your answer in another comment:

Reply:
`It's also possible when there is no comments.`